metrics/examples/prometheus/install/prometheus-rules.yaml

```diff
@@ -48,7 +56,46 @@ spec:
         severity: warning
       annotations:
         summary: 'Kafka offline log directories'
-        description: 'There are {{ $value }} offline log directoris on {{ $labels.kubernetes_pod_name }}'
+        description: 'There are {{ $value }} offline log directories on {{ $labels.kubernetes_pod_name }}'
+    - alert: ScrapeProblem
+      expr: up{job="kubernetes-cadvisor", kubernetes_name!~".+-cluster-zookeeper-client"} == 0
+      for: 3m
+      labels:
+        severity: major
+      annotations:
+        summary: 'Prometheus was unable to scrape metrics from {{ $labels.kubernetes_pod_name }}/{{ $labels.instance }}'
+        description: 'Prometheus has not been able to scrape metrics from {{ $labels.kubernetes_pod_name }}/{{ $labels.instance }} for more than 3 minutes'
+    - alert: ClusterOperatorContainerDown
+      expr: count((container_last_seen{container_name="strimzi-cluster-operator"} > (time() - 90))) < 1 or absent(container_last_seen{container_name="strimzi-cluster-operator"})
+      for: 1m
+      labels:
+        severity: major
+      annotations:
+        summary: 'Container strimzi-cluster-operator is down'
+        description: 'Container strimzi-cluster-operator has been down for more than 90 seconds'
+    - alert: KafkaBrokerContainerDown
+      expr: absent(container_last_seen{container_name="kafka",kubernetes_pod_name=~".+-kafka-[0-9]+"})
+      for: 3m
+      labels:
+        severity: major
+        group: ngp-kafka
+      annotations:
+        summary: 'Container kafka in pod with index 0 is down or in CrashLookBackOff status'
+        description: 'Container kafka in pod with index 0 is down or in CrashLookBackOff status for 3 minutes'
```

Comment:
The index seems to be variable - so we should remove it from the summary and description I guess.